chalice/app.py

```diff
@@ -17,15 +17,19 @@ def database_create_tables(event, context):
 
     db = DatabaseHandle()
 
-    table_list = []
+    try:
+        table_list = []
 
-    # created = True
-    for table_name in event['Tables']:
-        model = db.get_model(table_name)
-        table_list.append(model)
-        # model.create_table(safe=True)
+        # created = True
+        for table_name in event['Tables']:
+            model = db.get_model(table_name)
+            table_list.append(model)
+            # model.create_table(safe=True)
 
-    created = db.create_tables(table_list)
+        created = db.create_tables(table_list)
```

Comment:
what if we just do `created = db.create_tables(event['Tables'])` ?

Reply:
db.create_tables takes a list of Model classes. The event['Tables'] data is just a list of strings. The loop before goes and finds the corresponding Model class for each name in event['Tables']